package.json

```diff
@@ -37,7 +37,7 @@
     "get-stream": "^4.0.0",
     "git-log-parser": "^1.2.0",
     "nyc": "^13.1.0",
-    "semantic-release": "^15.0.0",
+    "semantic-release": "15.0.0",
```

Comment:
It should be `^15.0.0` not `15.0.0`. You should not change this line at all

Reply:
Sorry, coffee fully ingested now. I've added the caret back.